### PR TITLE
fix(x11/wine-stable): fix building after toolchain upgrade

### LINKED ISSUE #21926: [Bug]: `wine-stable` fails to be built in docker
### Problem description

Wine fails to be built in docker for arch `x86_64`.
Fresh docker instance did not help.
The problem is our toolchain, it can not work with `-mabi`.

```
echo | ~/.termux-build/_cache/android-r27b-api-24-v1/bin/x86_64-linux-android-clang -x c - -mabi=ms
clang: error: unsupported option '-mabi=' for target 'x86_64-unknown-linux-android24'
```

Last time package was updated at 22/Jan/24 and was not updated since.
But we updated our toolchain 3 times (26b, 27, 27b) so probably `-mabi` support was dropped in some of these releases.

### What steps will reproduce the bug?

```
./scripts/run-docker.sh ./build-package.sh -I -f -a x86_64 wine-stable
...
x86_64-linux-android-clang -m64 -c -o dlls/dinput/data_formats.o /home/builder/.termux-build/wine-stable/src/dlls/dinput/data_formats.c -Idlls/dinput \
  -I/home/builder/.termux-build/wine-stable/src/dlls/dinput -Iinclude \
  -I/home/builder/.termux-build/wine-stable/src/include \
  -I/home/builder/.termux-build/wine-stable/src/include/msvcrt -D_UCRT -D__WINESRC__ \
  -DDIRECTINPUT_VERSION=0x0700 -Wall -pipe -fcf-protection=none -fvisibility=hidden \
  -fno-stack-protector -fno-strict-aliasing -Wdeclaration-after-statement -Wempty-body \
  -Wignored-qualifiers -Winit-self -Wno-pragma-pack -Wstrict-prototypes -Wtype-limits \
  -Wunused-but-set-parameter -Wvla -Wwrite-strings -Wpointer-arith -fPIC \
  -fasynchronous-unwind-tables -D_WIN32 -fno-builtin -fshort-wchar -Wno-format -mabi=ms \
  -I/data/data/com.termux/files/usr/include 
clang: error: unsupported option '-mabi=' for target 'x86_64-unknown-linux-android24'
```
The error I mentioned happens with first invocation of `-mabi`, there are no other invocations mentioned in the log.
[wine-log.txt](https://github.com/user-attachments/files/17468851/wine-log.txt)

### PR DESCRIPTION
**Pull request by twaik**
Fixes #21926
Related to #21130

### CODE REVIEW

**Comment by twaik** [comment]
It is a simple fix, I'll merge it in 24 hours or after it gets 2+ approves.

**Review by licy183 — APPROVED** [review]

**Review by TomJo2000 — APPROVED** [review]

**Pull request merged**
